### PR TITLE
Add now playing card to top of app

### PR DESCRIPTION
**Pull request by woofers**
Adds a now playing card which shows which audio track is currently playing.  In the future we can show the labels obtained from the classifier here, as well as audio features and even a spectogram or graphs of other audio features. 

As a result now only 1 audio track can be played at a time.

now-playing

### CODE REVIEW

File: renderer/components/card.tsx [hunk 1]
```diff
@@ -9,18 +9,17 @@ const Card = ({
...
     title?: string;
     active?: boolean;
     href?: Url;
     onClick?: () => void;
     children?: React.ReactNode;
-    clickable?: boolean;
     className?: string;
 }) => {
     let activeStyle = {};
+    const clickable = typeof href != 'undefined' || typeof onClick === 'function';
```
Comment (julesyoungberg) [inline renderer/components/card.tsx:22]: nice :+1:

File: renderer/components/audio-player/audio-player.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const AudioPlayer = ({
+    sound,
+    waveColor: initialWaveColor,
+    progressColor: initialProgressColor,
+}: {
+    sound: Sound;
+    waveColor?: string;
+    progressColor?: string;
+}) => {
+    const { dispatch, state } = useAppState();
```
Comment (julesyoungberg) [inline renderer/components/audio-player/audio-player.tsx]: No need to destruct `state` here if it isn't used in this file.

**Review by julesyoungberg — APPROVED** [review]
this looks awesome man!

File: renderer/components/audio-player/audio-player.tsx (hunk 2 above)
Comment (woofers) [reply renderer/components/audio-player/audio-player.tsx]:
```suggestion
    const { dispatch } = useAppState();
```
Good catch, making a suggestion to fix this

**Comment by julesyoungberg** [comment]
merging this

**Pull request merged**